lib/Target/ExportSystemC/Patterns/SystemCEmissionPatterns.cpp

```diff
@@ -295,10 +354,23 @@ struct DynIntegerTypeEmitter : public TypeEmissionPattern<Ty> {
 
 void circt::ExportSystemC::populateSystemCOpEmitters(
     OpEmissionPatternSet &patterns, MLIRContext *context) {
-  patterns.add<SCModuleEmitter, SignalWriteEmitter, SignalReadEmitter,
-               CtorEmitter, SCFuncEmitter, MethodEmitter, ThreadEmitter,
-               SignalEmitter, InstanceDeclEmitter, BindPortEmitter,
-               AssignEmitter, VariableEmitter>(context);
+  // clang-format off
+  patterns.add<SCModuleEmitter,
+               SignalWriteEmitter,
+               SignalReadEmitter,
+               CtorEmitter,
+               SCFuncEmitter,
+               MethodEmitter,
+               ThreadEmitter,
+               SignalEmitter,
+               InstanceDeclEmitter,
+               BindPortEmitter,
+               AssignEmitter,
+               VariableEmitter,
+               NewEmitter,
+               DestructorEmitter,
+               DeleteEmitter>(context);
+  // clang-format on
```

Comment:
Nit: I'm usually against code format escape hatches unless there's strong motivation. One alternative is to break this up into logical groups with comments, e.g.:

```c++
patterns.add<
  Foo, Bar, Baz,
  // Variables
  Qux, Qux,
  // Operations
  Gorp
>
```

This is used in places like [`FIRRTLVisitors.h`](https://github.com/llvm/circt/blob/30bcfec8a66aa91b148c3df7d423867f8e9b82f1/include/circt/Dialect/FIRRTL/FIRRTLVisitors.h#L32).